Makefile

```diff
@@ -1,15 +1,12 @@
 UNAME := $(shell uname)
 
-transifex_langs = "ar,fr,es_419,zh_CN"
-i18n = ./src/i18n
-transifex_input = $(i18n)/transifex_input.json
```

Comment:
Please keep `transifex_input`. It's used on:

https://github.com/openedx/studio-frontend/blob/480b6d28adb8b03a4f7b0e58cd9b9856fd7b7ff1/Makefile#L74-L80